exporter/datasetexporter/README.md

```diff
@@ -0,0 +1,65 @@
+# DataSet Exporter
+
+<!-- status autogenerated section -->
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [development]   |
+| Supported pipeline types | logs   |
+| Distributions            | [contrib] |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
+
+This exporter sends logs to [DataSet](https://www.dataset.com/).
+
+See the [Getting Started](https://app.scalyr.com/help/getting-started) guide.
+
+## Configuration
+
+### Required Settings
+
+- `dataset_url` (no default): The URL of the DataSet API that ingests the data. Most likely https://app.scalyr.com. If not specified env variable `DATASET_URL` is used.
+- `api_key` (no default): The "Log Write" API Key required to use API. Instructions how to get [API key](https://app.scalyr.com/help/api-keys). If not specified env variable `DATASET_API_KEY` is used.
+
+### Optional Settings
+
+- `max_delay_ms` (default = "15000"): The maximum delay between sending batches from the same source.
+- `group_by` (default = []): The list of attributes based on which events should be grouped.
+- `retry_on_failure`: See [retry_on_failure](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md)
+- `sending_queue`: See [sending_queue](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md)
+- `timeout`: See [timeout](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md)
+
+
+### Example
+
+```yaml
+
+exporters:
+  dataset:
+    # DataSet API URL
+    dataset_url: https://app.scalyr.com
+    # API Key
+    api_key: your_api_key
+    # Send batch to the API at least every 15s
+    max_delay_ms: 15000
+    # Group data based on these attributes
+    group_by:
+      - attributes.container_id
+      - attributes.log.file.path
+      - body.map.kubernetes.container_hash
+      - body.map.kubernetes.pod_id
+      - body.map.kubernetes.docker_id
+      - body.map.stream
+
+service:
+  pipelines:
+    logs:
+      receivers: [otlp]
+      processors: [batch]
+      # add dataset among your exporters
+      exporters: [dataset]
+```
+
+[alpha]:https://github.com/open-telemetry/opentelemetry-collector#alpha
+[contrib]:https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
```

Comment:
you don't need those links down there anymore, since they're generated in the metadata status table.